src/components/Menu/index.tsx

```diff
@@ -25,6 +26,12 @@ const Menu = (props) => {
       setLang={setSelectedLanguage}
       cakePriceUsd={cakePriceUsd.toNumber()}
       links={config}
+      profile={{
+        username: profile?.username,
+        image: profile?.username ? `/images/nfts/${profile?.nft.previewImage}` : undefined,
```

Comment:
you probably can use `profile?.username ?? `/images/nfts/${profile?.nft.previewImage}``